admin/tabs/monitor/MonitorResultsModel.js

```diff
@@ -69,7 +69,9 @@ export class MonitorResultsModel {
         return XH
             .fetchJson({url: 'monitorAdmin/forceRunAllMonitors'})
             .then(
-                ToastManager.show({
+                // Wrapping the toast call in a function causes it to run only
+                // if the fetchJson was successful.
+                () => ToastManager.show({
```

Comment:
Good catch ensuring that this `then` takes an arrow function, but that is the standard form for these promise chains - it was just wrong before!  So no need for this comment here as it makes it seem like something unusual / in-particular is happening.